sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md

```diff
@@ -93,6 +93,19 @@ _The following was run on Windows 10 and Ubuntu Desktop 20.04 environments, with
 
     NOTE: If you are using WSL, do not run these commands from the Windows system drive (e.g. `/mnt/c/`).
 
+    ### Certificates - Important to know
+
+    For sample purposes, Embedded C SDK samples leverage certificates obtained when executing either `New-ArduinoZipLibrary.ps1` or `generate_arduino_zip_library.sh`. 
```

Comment:
AFAIK we don't pull the certificate when calling the Powershell script (yet). That is a bug reported by @ewertons . 
We can either choose to always require WSL on Windows (.sh script only) or come up with a way to download then transform the certificate into something loadable into the Arduino project (it might be possible to directly load a PEM file?). 

Currently the SH script is using `xxd` not available by default on Windows (easiest way to obtain it on Windows is to install the VIM editor).

Reply:
you're correct. just changed that to represent the current status.